cabal-testsuite/README.md

```diff
@@ -25,6 +25,25 @@ There are a few useful flags:
   the autodetection doesn't work correctly (which may be the
   case for old versions of GHC.)
 
+doctests
+========
+
+You need to install the doctest tool. Make sure it's compiled with your current
```

Comment:
This should address the GHC-version sensitivity.  It should at least be mentioned that if you switch GHC version, you need to reinstall `doctest`.  I switch GHC versions frequently.  Ideally I would not have to make "hard" switches (relinking `ghc` to `ghc-$VER`) but just use the `--with-compiler` flag.
I looked at a multi-GHC approach at:
- https://github.com/sol/doctest/issues/396.

Reply:
I thought that's exactly what I wrote here, but per your suggestion I reiterated it: please, checkout the new version.

Reply:
Thanks!